extension/chrome/elements/compose.ts

```diff
@@ -380,13 +387,21 @@ Catch.try(async () => {
     },
     lookupPubkeyFromDbOrKeyserverAndUpdateDbIfneeded,
     collectAllAvailablePublicKeys
-  }, processedUrlParams, await Store.subscription());
+  }, processedUrlParams);
 
   BrowserMsg.addListener('close_dialog', async () => {
     $('.featherlight.featherlight-iframe').remove();
   });
-  BrowserMsg.addListener('set_footer', async ({ footer }: Bm.SetFooter) => {
-    storage.email_footer = footer;
+  BrowserMsg.addListener('set_footer', async ({ email, footer }: Bm.SetFooter) => {
```

Comment:
There should not be any need for a set_footer BrowserMsg. It was used because there was a separate footer.htm frame, which is no longer needed. Since there is no possibility to set a different footer than what is on Gmail, this should be removed.

Reply:
Agree, removed